src/sultan/api.py

```diff
@@ -92,25 +90,16 @@ def load(cls, cwd=None, sudo=False, user=None, hostname=None, env=None, logging=
             context['user'] = 'root' if sudo else getpass.getuser()
         context.update(kwargs)
 
-        s = Sultan(context=context)
-        return s
+        return cls(context=context)
```

Comment:
Today I learned something new! It makes perfect sense, but I have been writing this the wrong way!